src/integration-test/java/de/danielbechler/diff/categories/CategoriesTestIT.groovy

```diff
@@ -0,0 +1,49 @@
+package de.danielbechler.diff.categories
+
+import de.danielbechler.diff.ObjectDifferBuilder
+import de.danielbechler.diff.introspection.ObjectDiffProperty
+import de.danielbechler.diff.node.DiffNode
+import de.danielbechler.diff.node.Visit
+import de.danielbechler.diff.path.NodePath
+import spock.lang.Specification
+
+class CategoriesTestIT extends Specification{
+
+	def obj1 = new MyObject("aaa","aaa", "aaa")
+	def obj2 =  new MyObject("bbb","bbb", "bbb")
+	def differ = ObjectDifferBuilder.startBuilding()
```

Comment:
Since there is only one test left, the setup can be moved into the test method, so it becomes self-contained.